receiver.go

```diff
@@ -66,6 +66,16 @@ func (r *Receiver) IssueCredit(credit uint32) error {
 	return nil
 }
 
+// DrainCredit sets the drain flag on the next flow frame and blocks until
+// the corresponding flow frame is received.
```

Comment:
Need a few more docs.
- calling DrainCredit() when a drain is in progress returns an error
- the state of the receiver if the context is canceled/expires

Reply:
Updated.